system/mp/mpcomm.cpp

```diff
@@ -1942,7 +1943,7 @@ CMPConnectThread::CMPConnectThread(CMPServer *_parent, unsigned port)
     mpSoMaxConn = 0;
     mpTraceLevel = 0;
     Owned<IPropertyTree> env = getHPCCEnvironment();
-    if (env)
+    if (env && parent->listen)
```

Comment:
also means:
```
 mpTraceLevel = env->getPropInt("EnvSettings/mpTraceLevel", 0);
```
is skipped, I know it's only used in run() currently, but I think a bit misleading not to set tracing level based on 'listen'.

Reply:
Good catch, thanks, I will resolve.